FreeBSD/usr/local/share/ceph/example/ceph-build-cluster.sh

```diff
@@ -0,0 +1,654 @@
+#!/bin/sh
```

Comment:
probably we can put this script into `src/script`? and install it `if(FREEBSD)`?